test/IRGen/abitypes_arm.swift

```diff
@@ -11,6 +11,6 @@ class Foo {
   }
 }
 
-// armv7: define internal void @makeOne(ptr noalias sret({{.*}}) align 4 %agg.result, float{{( noundef)?}} %f, float{{( noundef)?}} %s)
+// armv7: define internal void @makeOne(ptr{{( dead_on_unwind)?}} noalias{{( writable)?}} sret({{.*}}) align 4 %agg.result, float{{( noundef)?}} %f, float{{( noundef)?}} %s)
```

Comment:
Android armv7 generated `define internal void @makeOne(ptr dead_on_unwind noalias writable sret(%struct.One) align 4 %agg.result, float noundef %f, float noundef %s)` in its IR instead, so I added those attributes, optionally since I don't know if they will be there on other `armv7` platforms.

Reply:
It looks like Clang adds `dead_on_unwind` for `sret` args, so they always appear in a combination. I guess we don't need to make it optional.  https://github.com/swiftlang/llvm-project/blob/f1607d165cb61ef9f857d76afc6e8a04345bb41a/clang/lib/CodeGen/CGCall.cpp#L2622